src/image-marker.js

```diff
@@ -0,0 +1,56 @@
+import { $ } from 'clappr'
+import StandardMarker from './standard-marker'
+
+/*
+ * An implementation of an image Marker, which can show an image.
+ */
+export default class ImageMarker extends StandardMarker {
+
+  /*
+   * time: the time in seconds that this marker represents
+   * tooltipImage: the image to be shown (optional)
+   * width: width for the image (Default: 200px)
+   * height: height for the image (Default: auto)
+   */
+  constructor (time, tooltipImage, width, height) {
+    super(time, tooltipImage)
+    this._tooltipText = tooltipImage || null
```

Comment:
I think it would be nicer with another class `BaseMarker` which would basically be `StandardMarker` but without the `_buildMarkerEl()` function or `tooltipText`.

`StandardMarker` and this would then extend `BaseMarker` and add the `_buildMarkerEl()` implementation.

I can look at this tomorrow, if you want to leave this as is with the current name that's fine and I'll merge it?

Then I'll make the above and see if it works.